File: fizzbuzz.js
function fizzbizz(){
    let arr = []
    for(let i = 1; i<=100; i++){
        if(i%3 == 0 && i%5 == 0){
            arr.push("fizzbizz")
        }
        else if (i%3 == 0){
            arr.push("fizz")
        }
        else if (i%5 == 0){
            arr.push("bizz")
        }
        else{
            arr.push(i)
        }
        
    }
    console.log(arr)
}

fizzbizz()

File: fizzbuzz.rb
def fizzbuzz
 arr = []
for i in 1..100 do
    if i % 3 == 0 && i % 5 == 0
        arr.append("FizzBuzz")
        next
    end
    if i % 3 == 0
        arr.append("Fizz")
        next
    end
    if i % 5 == 0
        arr.append("Buzzzz")
        next
    end
    arr.append(i)
end
puts arr
end

fizzbuzz